src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -238,15 +241,15 @@ export default class DetailsBody extends React.Component<Props> {
                       environment={environment ? [environment] : undefined}
                       project={(projects as Project[]).map(project => Number(project.id))}
                       // TODO(davidenwang): allow interval to be changed for larger time periods
-                      interval="5m"
+                      interval="60s"
```

Comment:
I changed this to 60s for the sake of testing. Perhaps it is best that this matches the alert rule time bucket so that evaluation windows fall exactly on points in the graph. Could be a product discussion